src/Views/PartitioningView.vala

```diff
@@ -48,6 +50,24 @@ public class Installer.PartitioningView : AbstractInstallerView  {
         this.content_area.attach(description, 0, 0, 1, 1);
         this.content_area.attach(disk_scroller, 0, 1, 1, 1);
 
+        load_disks ();
+
+        gparted_button = new Gtk.Button.with_label (_("Modify Partitions"));
+        gparted_button.get_style_context ().add_class (Gtk.STYLE_CLASS_DESTRUCTIVE_ACTION);
```

Comment:
I do not know that having two destructive action buttons is good

Reply:
Yes

Reply:
It is done